### DISCUSSION TITLE
Volume storage utilization

### DISCUSSION THREAD

**Discussion by OS-andreineculai** [seaweedfs/seaweedfs#5817] [Q&A]
Hello, 
I'm new to this service. I have a setup with 1 Master, 3 Filer and 3 Volumes (10 Gb per volume) and a S3 for S3 API compatibile.
![image](https://github.com/user-attachments/assets/c3e401b3-e43f-4652-9c01-d5b516caf83b)
I'm only using S3 API to store files.
I don/t understand why each file uploaded take 1 GB of storage? The issue is that I'm running out of space. The file I'm testing has 7Mb in size.

**Comment by chrislusf** [comment]
https://github.com/seaweedfs/seaweedfs/wiki/FAQ#how-many-volumes-do-i-need
